content/docs/app-journey.md

```diff
@@ -95,6 +95,10 @@ deploying your new image to your favorite cloud!
 
 > In case you need it, `pack build` has a handy flag called `--publish` that will build your image directly onto a Docker
 > registry. You can learn more about `pack` features in the [documentation][pack-docs].
+>
+> Additionally, if the Docker daemon is using containerd storage and you export a container to the Docker daemon 
+> (building without `--publish`), it may degrade the performance of `pack build`. For more details, please refer to
+> [the related issue](https://github.com/buildpacks/pack/issues/2272).
```

Comment:
```suggestion
>
> Additionally, if your Docker daemon uses `containerd` image storage (not the Docker default),
> it may degrade the performance of `pack build`. For more details, please refer to
> [the related issue](https://github.com/buildpacks/pack/issues/2272).
```

Reply:
Folks who configure `containerd` are likely to be advanced users.  I've modified the text so that casual users can understand that this problem exists only in non-default configurations.

Reply:
I think your suggestion is better than my original phrasing. Thanks!

Reply:
It's worth noting, containerd will become the default eventually. Maybe "not the Docker default as of version x.x.x"?